cst/plugins/eu.esdihumboldt.cst.functions.groovy/src/eu/esdihumboldt/cst/functions/groovy/internal/GroovyUtil.java

```diff
@@ -305,6 +306,9 @@ public static Binding createBinding(InstanceBuilder builder, Cell cell, Cell typ
 		binding.setVariable(BINDING_INSTANCE_INDEX,
 				executionContext.getService(InstanceIndexService.class));
 
+		binding.setVariable(BINDING_LOOKUP_SERVICE,
+				executionContext.getService(LookupService.class));
```

Comment:
Do you think a specific binding is preferable over the more generic approach to add a binding for the `ServiceProvider`? (see [ING-4256](https://wetransform.atlassian.net/browse/ING-4256))

Reply:
No, if we can find a solution that avoids having to add individual bindings for every  service class that we want to use in Groovy scripts, that would be better IMO. Happy to close this PR in favour of a more generic approach!

Reply:
Would it be alright if we squash the last two commits?

Reply:
> Would it be alright if we squash the last two commits?

@emanuelaepure10 Yes, no need to keep my initial commit.